pkg/controller/controller.go

```diff
@@ -359,7 +359,7 @@ func (c *Controller) bootWithError(ctx context.Context) error {
 	// anything going wrong in our operators.
 	{
 		utilruntime.ErrorHandlers = []utilruntime.ErrorHandler{
-			func(_ context.Context, _ error, _ string, _ ...interface{}) {
+			func(_ context.Context, err error, msg string, _ ...interface{}) {
```

Comment:
Feels weird to have a mix of unamed and named arguments.

Reply:
yeah I copied from the original signature. I only rename err because is what we reused but [something fails in the tests](https://app.circleci.com/pipelines/github/giantswarm/operatorkit/1321/workflows/a1c0bb98-60d0-4092-a6b5-3dddc76920d1/jobs/14953) and I was trying to get more information